conf/machine/include/imx-base.inc

```diff
@@ -503,22 +503,18 @@ WKS_FILE_DEPENDS ?= " \
     ${@bb.utils.contains('MACHINE_FEATURES', 'optee', '${OPTEE_WKS_FILE_DEPENDS}', '', d)} \
 "
 
-WKS_FILE_DEPENDS:append:mx8-nxp-bsp  = " imx-boot"
-WKS_FILE_DEPENDS:append:mx8m-nxp-bsp = " imx-boot"
-
 # We need to restrict the append so we don't add this for other i.MX SoC's.
 # Derivatives that are not yet adopted the usage of boot container provided
 # by U-Boot build are still targeted to use 'imx-boot' package provided by
-# NXP. Moving those derivatives to mainline BSP would require to define an
+# NXP.
+#
+# Moving those derivatives to mainline BSP would require to define an
 # 'imx-boot-container' override, and test if the U-Boot built 'flash.bin'
 # binary is used a replacement.
-# Note, that the results binary name of the boot container is set to 'imx-boot'
+#
+# NOTE: the results binary name of the boot container is set to 'imx-boot'
 # for both NXP and Mainline BSP.
-# For Mainline BSP: the 'flash.bin' boot container is renamed during the
-# deployment task extesion execution defined in imx-boot-container class.
-# For NXP BSP: rename is done in 'imx-boot' recipe at the execution of compile
-# task.
```

Comment:
I'm unsure of removing this comment. It seems to me that it might still be helpfull

Reply:
We could have it inside the class as it doesn't belongs to `imx-base.inc` for sure.

Reply:
ok